cmd/mtaBuild.go

```diff
@@ -435,7 +435,7 @@ func getMtaID(mtaYamlFile string, utils mtaBuildUtils) (string, error) {
 
 	id, ok := result["ID"].(string)
 	if !ok || len(id) == 0 {
-		fmt.Errorf("Id not found in mta yaml file (or wrong type)")
+		return "", fmt.Errorf("Id not found in mta yaml file (or wrong type)")
```

Comment:
Is this somehow related?

Reply:
That was just something which I spotted on the road, an error being completely swapped away. I fixed it while considering it way to small for an extra PR.